ops/charm.py

```diff
@@ -519,8 +519,8 @@ def snapshot(self) -> Dict[str, Any]:
     def departing_unit(self) -> Optional[model.Unit]:
         """The :class:`ops.Unit` that is departing, if any.
 
-        You can use this to determine (for example) whether *you* are the
-        departing unit.
+        Use this to determine (for example) whether this unit is the
```

Comment:
I like the change, but I think we should say "Use this method ..." or similar for clarity.